### DISCUSSION TITLE
Shareable local config with json5

### DISCUSSION THREAD

**Discussion by bmuenzenmeyer** [renovatebot/renovate#10087] [Request Help] [answered]
I have a repo with two shareable configs:

![image](https://user-images.githubusercontent.com/298435/119047085-1343d900-b983-11eb-801d-06c9a6c47b0d.png)

Referencing `default.json` like this works:

```json5
{
  // all your base config is belong to us
  extends: ['local>BrianMuenzenmeyer/brian-renovate-config'],
}
```

> the org and repo exist in GHE I promise

But trying to reference the `json5` config does not:

```json5
{
  // all your base config is belong to us
  extends: ['local>BrianMuenzenmeyer/brian-renovate-config//renovate'],
}
```

> same result with `renovate.json5` btw, I did this because the docs state you don't need an extension.

![image](https://user-images.githubusercontent.com/298435/119047345-69b11780-b983-11eb-9621-919d8b87d026.png)

The [shareable config docs](https://docs.renovatebot.com/config-presets/#preset-hosting) for preset hosting mention `.json` - but I don't really know if this is two examples and more are supported.... or `.json` only. I don't get why `json5` would be supported in one location and not the other?

**Comment by rarkins** [comment, marked as answer]
Only JSON is supported for presets currently. See #7674
